Gemfile

```diff
@@ -40,6 +40,7 @@ gem 'bootstrap', '~> 4.0.0.beta3'
 gem 'devise_invitable', '~> 1.7.0'
 gem 'devise_account_expireable'
 gem 'mailgun-ruby'
+gem 'rubocop', '~> 0.54.0'
```

Comment:
You should probably move this into the `:development` group below. That way this gem won't be installed on our production server where it's not needed.